Surface_mesh/include/CGAL/Surface_mesh/Surface_mesh.h

```diff
@@ -1815,10 +1815,13 @@ class Surface_mesh
   
     template<class I, class T>
     std::pair<Property_map<I, T>, bool>
-    add_property_map(const std::string& name, const T t=T()) {
+    add_property_map(const std::string& name=std::string(), const T t=T()) {
+      if(name.empty()){
+        std::string name("anonymous-property-");
+        name += boost::lexical_cast<std::string>(anonymous_property_++);
```

Comment:
do we really want to include lexical_cast here rather than using `stringstream`?

Reply:
@sloriot  What is wrong about it?

Reply:
I'd better use stringstream rather than making Surface_mesh pulling `boost/lexical_cast.hpp`. @lrineau  do you agree?

Reply:
ping @lrineau @afabri

Reply:
I agree that there is no advantage here of `boost::lexical_cast`: the same can be done easily with standard C++ with only three lines instead of one:
```C++
std::stringstream ss;
ss << anonymous_property_++;
name += str.str();
```
or even the oneliner:
```C++
name += (std::stringstream() << anonymous_property_++).str();
```